terraform/test/main.tf

```diff
@@ -459,3 +459,9 @@ module "auth_internal_alb" {
   subnet_ids      = module.networking.auth_service_private_subnets.*.id
   vpc             = module.networking.vpc
 }
+
+module "open_id_connect" {
```

Comment:
Having this in both test and staging won't work, they're the same AWS account and we can't create the OIDC provider or roles twice. It should be in one or the other and that will need to support both.

Reply:
removed from staging